README.md

```diff
@@ -1,32 +1,239 @@
 ### [README still WIP, not contribution friendly yet]
 
-### Monitoring Server Configuration
-In `prometheus/prometheus.yml`, add hosts emitting metrics to the targets under scrape config's node job.
+Nodies Monitoring is a customizable and extensible monitoring solution for monitoring host machine metrics, container metrics, node status and node logs. 
+
+## Table of content
```

Comment:
We need a quick getting started guide that lets them set this up within 5-10 mins. The flow to set this up isn't necessarily clear. The readme breaks down the stack decently, but there needs to be a section for clueless people like me who just want to run some commands and set it up.